auto_round/utils.py

```diff
@@ -416,11 +416,16 @@ def get_multimodal_block_names(model, quant_vision=False):
     block_names = []
     target_modules = []
     vison_blocks_tuple = ("vision", "visual",)
+    last_block_name = ""
```

Comment:
Why is validate=True used in some cases and validate=False in others? If it's not due to time cost considerations, it doesn't make much sense.